src/commonMain/kotlin/generator/JsonSchemaGenerator.kt

```diff
@@ -334,3 +339,14 @@ private fun mapSchema(
 private inline fun <reified T : Annotation> List<Annotation>.find(): T? =
     filterIsInstance<T>()
         .firstOrNull()
+
+private fun getFallbackFormat(serialName: String): StringFormat? {
```

Comment:
amazing extension!

I think we can go on with it like this. Later I will work on unit testing and extend this list even more to also make JVM only tests. I will consider having this list extended depending on the platform.